build.gradle

```diff
@@ -97,7 +97,7 @@ dependencies {
     // kafka & zookeeper
     compile 'org.apache.kafka:kafka-clients:0.9.0.0'
     compile 'org.apache.kafka:kafka_2.11:0.9.0.0'
-    compile 'org.apache.zookeeper:zookeeper:3.4.5'
+    compile 'org.apache.curator:curator-framework:2.9.1'
```

Comment:
Yes, it's possible. Locally it works without exhibitor. Curator configured in the way that if exhibitor is not accessible it will use static zookeeper connection string as a fallback.